bodhi/server/util.py

```diff
@@ -343,6 +344,28 @@ def ci_status2html(context, status):
     return u"<span class='label label-%s'>Tests %s</span>" % (cls, status)
 
 
+def gating_status2html(context, status, reason=None):
+    """ Convert the specified status into a html string used to present the
+    gating status in a human friendly way.
+
+    Args:
+        context (mako.runtime.Context): Unused.
+        status (bool): A boolean value to indicate the current gating status.
+        reason (unicode): A short text explains the outcome of the gating.
+
+    Returns:
+        unicode: A human friendly HTML label of the gating status
+
+    """
+    if status:
+        return u'<span class="fa fa-check-circle text-success"></span>'
+    else:
+        html = u'<span class="fa fa-minus-circle text-danger"></span>'
+        if reason:
+            html += u'<a class="gating-summary" href="#automatedtests">%s</a>' % reason
```

Comment:
I just do not see any value of showing it as I think no one would care the summary if all the tests are passing.